content/docs/csidriver/troubleshooting/powerflex.md

```diff
@@ -6,6 +6,7 @@ description: Troubleshooting PowerFlex Driver
 
 | Symptoms | Prevention, Resolution or Workaround |
 |------------|--------------|
+| After installation vxflexos-node pods are in an `Init:CrashLoopBackOff` state in OpenShift 4.16 | Use SDC version 4.5.2.1 in OpenShift 4.16. |
```

Comment:
@rishabhatdell There could be many reasons besides the unsupported kernel version causing the CrashLoopBackOff issue in the older version. Would it be helpful to mention checking specific logs or similar issues we observed with the older SDC version on OCP 4.16?

What do you think?

Reply:
added